students/JoeNunnelley/session09/cli_main.py

```diff
@@ -0,0 +1,300 @@
+#! /usr/bin/env python3
+"""
+The Mailroom Program : Part 4 : Object Oriented
+Author : Joe Nunnelley
+Description:
+    A small command line based mailroom application designed to thank donors.
+"""
+import datetime
+import os
+import sys
+import tempfile
+import donor_models as d
```

Comment:
to make this a bit more readable, you may want to consider importing like:
from donor_models import Donor, DonorCollection

This way you can drop the d from before (just a minor cosmetic thing)